deployability/modules/allocation/README.MD

```diff
@@ -0,0 +1,444 @@
+                - track-output: "{working-dir}/agent-{agent}/track.yaml"
+          cleanup:
+            this: process
+            with:
+              path: python3
+              args:
+                - modules/allocation/main.py
+                - action: delete
+                - track-output: "{working-dir}/agent-{agent}/track.yaml"
+          foreach:
+            - variable: agents-os
+              as: agent
+      ```
+
+      Following the schema of the example:
+
+      Configure the following parameters depending on your test case:
+
+      ```yaml
+      variables/agent-os
+      variables/manager-os
+      infra-provider
+      working-dir
+      tasks
+      ```
+
+      Pay attention to the tasks:
+
+      ```yaml
+      args
+      depends-on
+      ```
+
+      >Note: In args, configure the launcher's path correctly (main.py files in each module), and to fill `depends-on`, consider the steps of your test (allocation, provision, and test)
+
+3. Execution of Command (local):
+
+  Execute the command by referencing the parameters required by the library (launcher).
+
+  ```bash
+  python3 -m workflow_engine {.yaml fixture path}
+  ```
+
+  Example
+
+  ```bash
+  python3 -m workflow_engine modules/workflow_engine/examples/dtt1-agents-poc.yaml
+  ```
+
+  > Note The command execution can also be mediated through Jenkins.
+
+#### Manual execution of the Allocation module
+
+If one wishes to execute the allocation module without installing the workflow engine, they can proceed by using the launcher ([module/allocation/main.py](main.py)):
+
+1. Create
+
+  While in wazuh-qa/deployability
+
+- Local deployment (Vagrant)
+
+  ```bash
+  python3 modules/allocation/main.py --action create --provider '{{ vagrant }}' --size '{{ large }}' --composite-name '{{ composite-name }}' --inventory-output '{{ inventory }}' --track-output '{{ track }}'
+
+  ```
+
+  Example:
+  ```bash
+  python3 modules/allocation/main.py --action create --provider vagrant --size large --composite-name linux-ubuntu-22.04-amd64 --inventory-output "/tmp/dtt1-poc/agent-linux-ubuntu-22.04-amd64/inventory.yaml" --track-output "/tmp/dtt1-poc/agent-linux-ubuntu-22.04-amd64/track.yaml"
+  ```
+
+- AWS deployment
+
+  ```bash
+  python3 modules/allocation/main.py --action create --provider '{{ aws }}' --size '{{ large }}' --composite-name '{{ composite-name }}' --inventory-output '{{ inventory }}' --track-output '{{ track }}' --label-termination-date '{{ termination-date }}' --label-team  '{{ team }}'
+
+  ```
+
+  >Note: In the case of AWS it is mandatory to define 2 arguments that are not necessary for Vagrant, --label-termination-date and --label-team.
+   --label-termination-date: This argument allows you to define the date on which the machine can be deleted. The allowed values are 1d (where the 1 refers to the number of days the machine is needed) or with the following format "2024-03-20 21:00:00"
```

Comment:
Add emphasis to 1, we need to make sure that we are referring to the value, and apply this to the entire document

> are `1d` (where the `1` refers

Reply:
> The allowed values are **1d** (where the **1** refers to the number of days the machine is needed) or with the following format "2024-03-20 21:00:00"

or something like

> The allowed values are '1d' (where the '1' refers to the number of days the machine is needed) or with the following format "2024-03-20 21:00:00"

Reply:
Done a81aec813